scripts/storage-emulator-integration/tests.ts

```diff
@@ -1212,6 +1212,32 @@ describe("Storage emulator", () => {
 
         expect(error).to.contain("does not exist.");
       });
+
+      it("delete prevented by security rule", async () => {
+        const downloadUrl = await page.evaluate((filename) => {
+          return firebase.storage().ref(filename).getDownloadURL();
+        }, filename);
+
+        expect(downloadUrl).to.be.not.null;
```

Comment:
I know the rest of this file doesn't do a good job of this but please test one thing per test case. See [go/unit-test-practices#behavior-testing](https://goto.google.com/unit-test-practices#behavior-testing)

Reply:
Same comment for `#delete()` test case above since we're already here

Reply:
Ok yeah, I realised there's no need to test `getDownloadURL` here since there's already a test for it above. Removed from both delete tests.